### PR TITLE
qa/suites/rados: do not test with el7

### PR DESCRIPTION
**Pull request by tchaikov**
since we stopped building master on el7, there is no need to test
cephadm with el7 anymore.

Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
this change depends on https://github.com/ceph/teuthology/pull/1520

File: qa/suites/rados/thrash-old-clients/ceph.yaml [hunk 1]
```diff
@@ -1,2 +1,3 @@
+verify_ceph_hash: false
```
Comment (sebastian-philipp) [inline qa/suites/rados/thrash-old-clients/ceph.yaml:1]:
```suggestion
verify_ceph_hash: false  # Don't verify os + flavor + sha1
```

I found the name `hash` a bit misleading.

Comment (tchaikov) [reply qa/suites/rados/thrash-old-clients/ceph.yaml:1]: yes, "hash" is more related to how this function is implemented -- it query shaman with the sha1 of the ceph being tested for its packages. and as you put sha1 is only one of the parameters of the query. will incorporate your change.

Comment (tchaikov) [reply qa/suites/rados/thrash-old-clients/ceph.yaml:1]: also, i want the name align with its counterpart "suite_verify_ceph_hash" in teuthology. so one can grep / search it for more context in future.

Comment (tchaikov) [reply qa/suites/rados/thrash-old-clients/ceph.yaml:2]: updated and repushed.

**Comment by sebastian-philipp** [comment]
While looking into https://tracker.ceph.com/issues/46529, it turned out that this PR conflicts with #32377 

* #35719 removes centos_7 from suites/rados/cephadm
* #32377 test suites/rados/thrash-old-clients only centos_7

Right now, this conflict makes `suites/rados/thrash-old-clients´ the only suite that tests cephadm on Centos 7. And it turns out that we likely have a problem with podman on CentOS 7.6. Thus I see two options:

1. We revert this PR and continue to test cephadm on CentOS 7. Then, we'd need someone with in-depth podman experience to debug the issue we see with CentOS 7.6.
3. Alternatively, we revert #32377 and test `thrash-old-clients` using the traditional package based deployment. 

@tchaikov @jdurgin @liewegas . This depends a bit on if you see the need to support cephadm on Centos 7.

**Comment by tchaikov** [comment]
i think this change was originated from a cleanup to drop el7 bits from `ceph.spec.in` and `install-deps.sh` as i learned from @joshd that we can stop building ceph on el7, which led to a series of changes

- stop building master on el7
- stop testing master on el7

i am not sure what "conflict" here stands for. do you mean we are using a combination of cephadm+el7 for testing which is not covered by our testing anymore?

**Comment by sebastian-philipp** [comment]
to recap, the conflict is: #32377  still tests cephadm on CentOS 7 in suites/rados/thrash-old-clients, despite this PR here removed the test on CenOS 7 for rados/cephadm. 

Things should still be fine, but unfortunately they aren't: I'm seeing test failures in `suites/rados/thrash-old-clients` which I cannot reproduce in `suites/rados/cephadm`. And they **might** be caused due to the older kernel of CentOS 7. 

If we don't need cephadm on CenOS 7, then we should revert #32377 to use the traditional deployment for `suites/rados/thrash-old-clients`

**Comment by sebastian-philipp** [comment]
https://github.com/ceph/ceph/pull/36321 is the revert of #32377

**Pull request merged**